### PR TITLE
Added support for parsing an ODBC Literal Escape Sequence. Created Unit ...

### PR DESCRIPTION
**Pull request by jheimbouch**
Hello, in regards to Issue https://github.com/NLPchina/elasticsearch-sql/issues/45, I have implemented a fix to allow parsing for an ODBC Literal Escape Sequence. Link to MSDN here: https://msdn.microsoft.com/en-gb/library/windows/desktop/ms712360(v=vs.85).aspx

I wrote a unit test, that passed, and ensured all others did as well.

### CODE REVIEW

File: src/test/java/org/nlpcn/es4sql/MainTestSuite.java [hunk 1]
```diff
@@ -128,7 +150,7 @@ private static InetSocketTransportAddress getTransportAddress() {
 		String port = System.getenv("ES_TEST_PORT");
 
 		if(host == null) {
-			host = "localhost";
+			host = "10.1.1.44";
```
Comment (omershelef) [inline src/test/java/org/nlpcn/es4sql/MainTestSuite.java]: Please fix it back to localhost

**Comment by omershelef** [comment]
Thank you from the contribution. please fix the host back to "localhost" instead of "10.1.1.44" in MainTestSuite

**Comment by jheimbouch** [comment]
@omershelef, I have changed it back. Apologies, I missed that when originally committing.

File: .travis.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ jdk:
 
 
 before_install:
-  - wget https://download.elasticsearch.org/elasticsearch/elasticsearch/elasticsearch-1.4.1.deb && sudo dpkg -i --force-confnew elasticsearch-1.4.1.deb
+  - wget https://download.elasticsearch.org/elasticsearch/elasticsearch/elasticsearch-1.4.1.deb && sudo dpkg -i --force-confnew elasticsearch-1.4.1.deb --no-check-certificate
```
Comment (omershelef) [inline .travis.yml]: Please remove --no-check-certicicate.

**Comment by omershelef** [comment]
The tests when running in java 8 are still failing, if you have time, it will be nice to fix it.
you can see the details here: https://travis-ci.org/NLPchina/elasticsearch-sql/builds/55103049

**Comment by jheimbouch** [comment]
I will look into this when I get back to the office. Thanks!

**Comment by jheimbouch** [comment]
@omershelef, I ran 'mvn package' on my localhost, with jdk1.8.0_40 and the build passed.

I see that the TravisCI is building with 1.8.0_31. Is it possible to get that updated?

**Pull request merged**